### PR TITLE
Decouple menu from app

### PR DESCRIPTION
**Pull request by mkoistinen**
No longer automatically attach a menu with the app. Since we offer a choice, the user should attach the desired menu (if any).

### CODE REVIEW

File: docs/introduction/basic_usage.rst [hunk 1]
```diff
@@ -2,11 +2,17 @@
 Basic usage
 ###########
 
-Create a django CMS page to hook the Aldryn Jobs application into. In its *Advanced settings*,
-set its ``Application`` to *Jobs*, and use the provided, default *Jobs / aldryn_jobs* in
-``Application configurations`` before saving.
+Create a django CMS page to hook the Aldryn Jobs application into. In its
+*Advanced settings*, set its ``Application`` to *Jobs*, and use the provided,
```
Comment (evildmp) [inline docs/introduction/basic_usage.rst]: that comma after provided should be excised

File: docs/introduction/basic_usage.rst [hunk 2]
```diff
@@ -2,11 +2,17 @@
...
+default *Jobs / aldryn_jobs* in ``Application configurations`` before saving.
 
-The page is now a landing page for job openings; it will be empty until you create some.
+The page is now a landing page for job openings; it will be empty until you
+create some.
 
-The behaviour of the Jobs system should be largely self-explanatory, but the :doc:`tutorial for
-users </user/index>` will guide you through some basic steps if necessary.
+Also in *Advanced settings*, you may choose to attach one of two menus as a
+sub-menu to the page in the site navigation: the ``Job Categories Menu`` or
```
Comment (evildmp) [inline docs/introduction/basic_usage.rst]: If "Job Categories Menu" is a field value, it should be *emphasised* - http://docs.django-cms.org/en/develop/contributing/contributing.html#inline-markup

**Comment by evildmp** [comment]
Looks good @mkoistinen other than my minor notes

**Comment by Chive** [comment]
LGTM

File: docs/introduction/basic_usage.rst (hunk 2 above)
Comment (mkoistinen) [inline docs/introduction/basic_usage.rst]: :+1:

File: docs/introduction/basic_usage.rst (hunk 1 above)
Comment (mkoistinen) [inline docs/introduction/basic_usage.rst]: :+1:

**Pull request merged**